services-core/src/gas_price/gnosis_safe.rs

```diff
@@ -59,34 +58,30 @@ const SAFE_LOW_PERCENTILE: f64 = 0.3;
 // price can be 1000 times the fast gas price which is not reasonable.
 const SECONDS_PER_BLOCK: f64 = 15.0;
 // Treat percentiles as probabilities for geometric distribution.
-const FAST_TIME: f64 = SECONDS_PER_BLOCK * FAST_PERCENTILE;
-const STANDARD_TIME: f64 = SECONDS_PER_BLOCK * STANDARD_PERCENTILE;
-const SAFE_LOW_TIME: f64 = SECONDS_PER_BLOCK * SAFE_LOW_PERCENTILE;
+const FAST_TIME: f64 = SECONDS_PER_BLOCK / FAST_PERCENTILE;
+const STANDARD_TIME: f64 = SECONDS_PER_BLOCK / STANDARD_PERCENTILE;
+const SAFE_LOW_TIME: f64 = SECONDS_PER_BLOCK / SAFE_LOW_PERCENTILE;
```

Comment:
This should probably be its own PR, but can you double check that I'm not crazy and this logic was off? The ignored test was failing.

Reply:
Yes you're right. I checked the original PR. It was initially correct and then in https://github.com/gnosis/dex-services/pull/1500/commits/90d4fc130269323e13861913016476c7c249db0f#diff-3af02e6a043d788af05800f15cbd96e304d2eb57faa2c56d3d17e6d78aff8937R64 I got rid off the unnecessary `1.0 / x` and accidentally made it a multiplication :facepalm: .

Reply:
:see_no_evil:

Reply:
Isolated the fix in #1526